src/Powercord/coremods/moduleManager/commands/share.js

```diff
@@ -0,0 +1,121 @@
+const { getModule, constants: { ComponentActions } } = require('powercord/webpack');
+const fs = require('fs');
+const path = require('path');
+const { resp } = require('../util');
+
+const { openURL } = getModule([ 'openURL' ], false);
+const { ComponentDispatch } = getModule([ 'ComponentDispatch' ], false);
+
+function formatUrl (url) {
+  return url
+    .replace('.git', '')
+    .replace('git@github.com:', 'https://github.com/')
+    .replace('url = ', '');
+}
```

Comment:
Let's make a shared util to get git into of a plugin. Can also include things like the branch, but don't worry about that for now. Just add a property for the web URL.

Reply:
What?

Reply:
I think he means to make it a separate utility function.

Reply:
Yeah, create a function in the coremod utils folder that takes a plugin/theme ID import and the type and returns an object with some git-related info. For now, just the web URL is fine, but in the future we can add branch, revision, etc.